roles/smartgateway/defaults/main.yml

```diff
@@ -3,3 +3,5 @@
 size: 1
 container_image_path: quay.io/redhat-service-assurance/smart-gateway:latest
 prometheus_scrape_interval: 1s
+use_basic_auth: false
```

Comment:
Should we default this to true now; or at least soon after all the ECK code merges?

Reply:
Well, the defaults here are basically for non-SAO managed stand ups (at least that's what I've been thinking in my head). If someone were to stand up a SG (for some strange reason) independent of the SAO, I would somewhat expect it to "just work" (which is not a tested scenario currently).

The SAO will pass all values it requires to the SGO via the `SmartGateway` object, so will override this to `use_basic_auth: true`. You can see that here:  https://github.com/redhat-service-assurance/service-assurance-operator/pull/27/files#diff-20856cacc013bbb9e77b19cced6ac91aR28

Basically, think of the defaults here as non-SAF defaults.